src/stp/evaluations/game_states/TimeOutGameStateEvaluation.cpp

```diff
@@ -0,0 +1,14 @@
+//
```

Comment:
What happened to all these files for them to be marked as new? Clearly, they're just copies from old versions.

Reply:
refactor of invariants to evaluations